typescript/infra/config/environments/mainnet3/warp/TIA-eclipsestride-deployments.yaml

```diff
@@ -0,0 +1,23 @@
+# Configs and artifacts for the deployment of Hyperlane Warp Routes
+description: Hyperlane Warp Route artifacts
+timestamp: '2024-10-18T14:00:00.000Z'
+deployer: Abacus Works (Hyperlane)
+data:
+  config:
+    stride:
+      protocolType: cosmos
+      type: collateral
+      hypAddress: stride134axwdlam929m3mar3wv95nvkyep7mr87ravkqcpf8dfe3v0pjlqwrw6ee
```

Comment:
I think this is the stTIA address, should be `stride1pvtesu3ve7qn7ctll2x495mrqf2ysp6fws68grvcu6f7n2ajghgsh2jdj6`